website1/index.html

```diff
@@ -1,71 +1,102 @@
+          </li>
+          <li class="menu__link">
+            <a href="#"> Автономність </a>
+          </li>
+          <li class="menu__link">
+            <a href="#"> Етапи роботи </a>
+          </li>
+          <li class="menu__link">
+            <a href="#"> Контакти </a>
+          </li>
+        </ul>
+        <a href="#" class="language">UA</a>
+        <button class="btn_call">Зателефонуйте мені</button>
+      </div>
+    </div>
+  </header>
+  <section id="first-page" class="first-page section">
+    <div class="container mx-auto">
+      <div class="first-page__content">
+        <p class="first-page__content-item first-page__text_narrow">
+          БУДІВНИЦТВО ІНДИВІДУАЛЬНОГО
+        </p>
+        <p class="first-page__content-item first-page__text_wide">
+          ХРАНИЛИЩА
+        </p>
+        <p class="first-page__content-item first-page__text_default20">
+          Безпека під час бойових дій та стихийних лих
+        </p>
+        <div class="button_active_with_shadow">
+          <button class="first-page__content-item button_active">
+            Замовити консультацію
+          </button>
+          <div class="button_active__shadow"></div>
         </div>
-      </section>
-      <footer id="footer" class="footer">
-        <div class="container"></div>
-      </footer>
+      </div>
+    </div>
+  </section>
+  <section class="appointment_bunker">
+    <div class="title__section">
+      <h1 class="title__header">ПРИЗНАЧЕННЯ БУНКЕРА</h1>
+      <hr class="title__line" size="4px" width="190px" color="#FFC300">
+      <p class="appointment__header-text">Власний бункер запезпечить Вам відчуття безпеки і у момент небезпечних
+        ситуацій <b class="bold-text">врятує Вас</b> і Вашу родину від:</p>
+    </div>
+    <div class="box-threats">
+      <div class="threats war">
+        <img src="./assets/icons/bullets.svg" alt="icon bullets" class="icons-threats">
+        <p class="name-threat">Повномасштабних воєнних дій</p>
+        <p class="protection-description">
+          Захищає від стрілецької зброї - автоматів, кулеметів та гарматних снарядів малого та середнього калібрів.
+        </p>
+      </div>
+      <div class="threats cataclysms">
+        <img src="./assets/icons/tornado.svg" alt="icon tornado" class="icons-threats">
+        <p class="name-threat">Природних катаклізмів</p>
+        <p class="protection-description">
+          Захистить від ураганів, аномальних дощів, гроз та великого граду. Також від занадто високої температури повітря, під час пожеж
+        </p>
+      </div>
+      <div class="threats pollution">
+        <img src="./assets/icons/nuclear-reactor.svg" alt="icon nuclear-reactor" class="icons-threats">
+        <p class="name-threat">Забруднення повітря</p>
+        <p class="protection-description">
+          Якщо повітря небезпечне для дихання у наслідок лісових пожеж та промислових отруйгих речовин у атмосферу
+        </p>
+      </div>
```

Comment:
It's block from `BEM`, see more information on how to write a name for the **inner element** and rebase class names.